### DISCUSSION TITLE
[Desktop] Stretch(large) Video Call Window During Chat

### DISCUSSION THREAD

**Discussion by ghost** [wireapp/wire-webapp#11156] [Ideas]
wire-desktop application within start video call able see large screen video call window properly, however when open chat window then video screen become very smaller as useless.

------------

wire screen - 1
![screenshot_2](https://user-images.githubusercontent.com/45872460/50119317-050aa380-0285-11e9-896d-0a5740e3e117.png)

wire screen - 2
![screenshot_1](https://user-images.githubusercontent.com/45872460/50119348-194ea080-0285-11e9-8ab7-ef2e393ce57f.png)

-------------

below screenshot about Skype-Desktop client (windows) 

![screenshot_3](https://user-images.githubusercontent.com/45872460/50119970-d2fa4100-0286-11e9-8958-f6a8b34dab7f.png)

---------

basically that isnt hard task to develop however it may more helpful for every users.
please enhance it within next release.^^

**Comment by itsnotsaved** [comment]
I was tracking this feature request.
@bennycode @ffflorian did you close this without implement? using 3.26.2941 | 2021.06.01.12.23 but still happen [wire screen - 2] when open chat.

**Reply by ffflorian** [reply]
We moved all feature requests to discussions and are using the issues only for bugs.
